test/functionalTest/cases/1677_geo_location_for_api_v2/ngsiv2_location_in_ngsv1_near.test

```diff
@@ -0,0 +1,391 @@
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+NGSIv2 geo-queries as payload in NGSIv1 queryContext ('near' cases)
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+# Inspired by 0000_ngsi10_geolocation_query tests
+#
+# 01. Create entity corresponding to Madrid
+# 02. Create entity corresponding to Alcobendas
+# 03. Create entity corresponding to Leganes
+# 04. Query inside 13.5 kms radius from Madrid center -> Madrid, Leganes
+# 05. Query inside 15 kms radius from Madrid center  -> Madrid, Alcobendas, Leganes
+# 06. Query in 13.5 kms radius outside Madrid center -> Alcobendas
+
+echo "01. Create entity corresponding to Madrid"
+echo "========================================="
+payload='{
+  "id": "Madrid",
+  "type": "City",
+  "location": {
+    "value": "40.418889, -3.691944",
+    "type": "geo:point"
+  }
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+sleep 1.1
+
+
+echo "02. Create entity corresponding to Alcobendas"
+echo "============================================="
+payload='{
+  "id": "Alcobendas",
+  "type": "City",
+  "location": {
+    "value": "40.533333, -3.633333",
+    "type": "geo:point"
+  }
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+sleep 1.1
+
+
+echo "03. Create entity corresponding to Leganes"
+echo "=========================================="
+payload='{
+  "id": "Leganes",
+  "type": "City",
+  "location": {
+    "value": "40.316667, -3.75",
+    "type": "geo:point"
+  }
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "04. Query inside 13.5 kms radius from Madrid center -> Madrid, Leganes"
+echo "======================================================================"
+payload='{
+  "entities": [
+  {
+    "type": "City",
+    "isPattern": "true",
+    "id": ".*"
+  }
+  ],
+  "restriction": {
+    "scopes": [
+    {
+      "type" : "FIWARE::Location::NGSIv2",
+      "value" : {
+        "georel": [ "near", "maxDistance:13500", "minDistance:0" ],
+        "geometry": "point",
+        "coords": [ [40.418889,-3.691944] ]
+      }
+    }
+    ]
+  }
+}'
+orionCurl --url /v1/queryContext --payload "$payload" --json
+echo
+echo
+
+
+echo "05. Query inside 15 kms radius from Madrid center -> Madrid, Alcobendas, Leganes"
+echo "================================================================================"
+payload='{
+  "entities": [
+  {
+    "type": "City",
+    "isPattern": "true",
+    "id": ".*"
+  }
+  ],
+  "restriction": {
+    "scopes": [
+    {
+      "type" : "FIWARE::Location::NGSIv2",
+      "value" : {
+        "georel": [ "near", "maxDistance:15000" ],
```

Comment:
As far as I understand, changoing = to : in maxDistance should be also changes in NGSIv2 tests, i.e.

georel=near;maxDistance=1500

should change to

georel=near;maxDistance:1500

However, no trace of these changes in this PR...

In addition, the NGSIv2 .apib should be changed as current version describes maxDistance and minDistance with = instead of :